packages/react-native-reanimated/src/ReanimatedModule/NativeReanimated.ts

```diff
@@ -99,38 +103,38 @@ See https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooti
     if (__DEV__) {
       checkCppVersion();
     }
-    this.InnerNativeModule = global.__reanimatedModuleProxy;
+    this.#reanimatedModuleProxy = global.__reanimatedModuleProxy;
```

Comment:
Since we're assigning `global.__reanimatedModuleProxy` to `this.#reanimatedModuleProxy`, does it make sense to reset `global.__reanimatedModuleProxy` to undefined here?

Reply:
Leaving it as is as we discussed offline.